docs/releases/4.2.1.txt

```diff
@@ -33,3 +33,7 @@ Bugfixes
 * Fixed a regression in Django 4.2 where creating copies and deep copies of
   ``HttpRequest``, ``HttpResponse``, and their subclasses didn't always work
   correctly (:ticket:`34482`, :ticket:`34484`).
+
+* Fixed a regression in Django 4.2 where ``timesince()`` would return incorrect
```

Comment:
`timeuntil` is also affected:

_"Fixed a regression in Django 4.2 where ``timesince`` and ``timeuntil`` template filters would ..."_